lib/ts/recipe/totp/components/themes/mfa/totpCodeVerificationFooter.tsx

```diff
@@ -21,23 +21,14 @@ import type { TOTPMFACommonProps } from "../../../types";
 export const CodeVerificationFooter = withOverride(
     "TOTPCodeVerificationFooter",
     function TOTPCodeVerificationFooter({
-        showFactorChooserButton,
         onSignOutClicked,
-        onFactorChooserButtonClicked,
     }: TOTPMFACommonProps & {
-        showFactorChooserButton: boolean;
-        onFactorChooserButtonClicked: () => void;
         onSignOutClicked: () => void;
     }): JSX.Element {
         const t = useTranslation();
 
         return (
             <div data-supertokens="footerLinkGroupVert totp-mfa codeVerificationFooter">
-                {showFactorChooserButton && (
-                    <div data-supertokens="secondaryText" onClick={onFactorChooserButtonClicked}>
-                        {t("TOTP_MFA_FOOTER_CHOOSER_ANOTHER")}
-                    </div>
-                )}
```

Comment:
why have we removed this?

Reply:
You asked to remove the choose another factor buttons and use the back arrow instead.